src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -172,14 +172,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 
 				$entityId = $dataValue->getEntityId();
 
-				if ( !( $entityId instanceof Int32EntityId ) ) {
+				if ( !( $entityId instanceof ItemId ) ) {
```

Comment:
i see that stuff is called qid in buildTupleCondition but are we sure we don't have or can support entity id values that have property ids here?